using_apis.md

```diff
@@ -158,6 +158,21 @@ And voila! <br/>
 We have successfully parsed our json within kotlin. <br/>
 One thing to note is that you don't need to add all of the json key/value pairs to the structure, you can just have what you need.
 
+**Shell**
+Here is how you could extract the values for the different keys in the json, using sed and tr:
+
+1) Extract the `climate` value:
+```sh
+curl "https://swapi.dev/api/planets/1/" | sed -nE "s/.*\"climate\":\"([^\"]*)\".*/\1/p" # note that we are using the [^\"]* pattern as a replacement for greedy matching in standard regex, as posix sed does not support greedy matching; we are also escaping the quotation mark
```

Comment:
With a complex regex like this it might be nice to have a screenshot from regex101.com or something similar to highlight how the regex works. Many people will just see what looks like random letters and get confused :pensive: